runners/tez/src/main/java/org/apache/beam/runners/tez/translation/GroupByKeyTranslator.java

```diff
@@ -0,0 +1,26 @@
+package org.apache.beam.runners.tez.translation;
+
+import com.google.common.collect.Iterables;
+import org.apache.beam.sdk.transforms.GroupByKey;
+import org.apache.beam.sdk.values.PValue;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * {@link GroupByKey} translation to Tez {@link org.apache.tez.runtime.library.conf.OrderedPartitionedKVEdgeConfig}
+ */
+class GroupByKeyTranslator<K, V> implements TransformTranslator<GroupByKey<K, V>> {
+  private static final Logger LOG = LoggerFactory.getLogger(GroupByKey.class);
+
+  @Override
+  public void translate(GroupByKey<K, V> transform, TranslationContext context) {
+    if (context.getCurrentInputs().size() > 1 ){
+      throw new RuntimeException("Multiple Inputs are not yet supported");
```

Comment:
<!--new_thread; commit:346274d62540a16c9bf795fc6e9ab2cd28430008; resolved:0-->
Seems like these should be NotImplementedException or UnsupportedOperationException?